src/static/js/pad.js

```diff
@@ -280,6 +280,12 @@ function handshake()
         $('#options-chatandusers').parent().hide();
         $('#options-stickychat').parent().hide();
       }
+      
+      
+      if(clientVars.padOptions['useLoginForName'] && clientVars.userName!= null){
```

Comment:
Please fix style here: `clientVars.userName!= null` (add spaces; use `!==` instead of `!=`)